camel/memories/records.py

```diff
@@ -45,6 +47,9 @@ class MemoryRecord(BaseModel):
     role_at_backend: OpenAIBackendRole
     uuid: UUID = Field(default_factory=uuid4)
     extra_info: Dict[str, str] = Field(default_factory=dict)
+    timestamp: float = Field(
+        default_factory=lambda: datetime.now().timestamp()
```

Comment:
would use UTC time instead of local time be more robust? Same as other parts
```suggestion
        default_factory=lambda: datetime.now(timezone.utc).timestamp()
```

Reply:
done